robottelo/cli/factory.py

```diff
@@ -83,10 +84,10 @@ def create_object(cli_object, options, values):
 
     """
     update_dictionary(options, values)
-    result = cli_object.create(options)
-
-    # If the object is not created, raise exception, stop the show.
-    if result.return_code != 0:
+    try:
+        result = cli_object.create(options)
+    except CLICommandExecutionError:
```

Comment:
This can be discussed, too. I left `CLIFactoryError` here for compatibility reasons, but it can be dropped easily as all the tests will be refactored and `CLIFactoryError` can be replaced with `CLICommandExecutionError`. Should I leave it as it is or drop?